README.md

```diff
@@ -825,7 +825,7 @@ This repository also provides one such platforms where contributers come over an
 |[Wheel_of_Fortunes](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Wheel_of_Fortunes)|
 |[Tic-tac-toe](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Tic-tac-toe)|
 |[Quest_For_Riches](https://github.com/kunjgit/GameZone/tree/main/Games/Quest_For_Riches)|
-
+|[Pattern Creation Game](https://github.com/ananyag309/GameZone_ggsoc/tree/main/Games/Pattern_Creation_Game)|
```

Comment:
change your name with the repository name as other did eg kunjgit/GameZone

Reply:
done